docs/install/containers/images/backend.md

```diff
@@ -304,6 +311,34 @@ And start a container.
 docker run -p 8080:8080 myproject:latest start
 ```
 
+### Adding `zope.conf` configuration fragments
+
+In the directory containing your `Dockerfile`, create a folder `etc/zope.conf.d`.
+Add your `zope.conf` configuration fragments there.
+
+Now add the following to your `Dockerfile`:
```

Comment:
Where should it be added? Without guidance, some people will place it in an unfortunate point in the Dockerfile and not take advantage of skipping over steps that have not changed.

Reply:
Good point.  Will fix.